### PR TITLE
Add joins for broker, ewallet, derm_treatment, car_dealership

### PR DESCRIPTION
**Pull request by wendy-aw**

### CODE REVIEW

**Review by rishsriv — COMMENTED** [review]
Thanks Wendy! Sorry I'm taking a while to review this -- wanted to triple check that the joins are all valid and will then merge!

File: defog_data/supplementary.py [hunk 1]
```diff
@@ -545,8 +545,86 @@ def load_embeddings(emb_path: str) -> tuple[dict, dict]:
...
+        ],
+        ("consumer_div.merchants", "consumer_div.coupons"): [
+            ("consumer_div.merchants.mid", "consumer_div.coupons.merchant_id")
+        ],
+        ("consumer_div.merchants", "consumer_div.wallet_merchant_balance_daily"): [
+            ("consumer_div.merchants.mid", "consumer_div.wallet_merchant_balance_daily.merchant_id")
+        ],
+        ("consumer_div.coupons", "consumer_div.wallet_transactions_daily"): [
+            ("consumer_div.coupons.cid", "consumer_div.wallet_transactions_daily.coupon_id")
+        ]
```
Comment (wongjingping) [inline defog_data/supplementary.py]: 2 more to add:
`consumer_div.coupons.merchant_id` can be joined with `consumer_div.wallet_transactions_daily.sender_id` and `consumer_div.wallet_transactions_daily.receiver_id`

**Review by wongjingping — APPROVED** [review]
Thank you for the careful and detailed additions here that will enable testing of join hints for the basic/advanced schema!

**Comment by rishsriv** [comment]
Thanks the the suggestion JP! Committed and fixed. Everything else looks good, too!

**Comment by wendy-aw** [comment]
Thanks for the sharp eyes and fixes!

**Review by rishsriv — APPROVED** [review]

**Pull request merged**